src/main/java/ch/njol/skript/effects/EffItemLifetime.java

```diff
@@ -0,0 +1,62 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item Despawn")
+@Description("Prevent a dropped item from naturally despawning through Minecraft's timer.")
+@Examples({
+	"prevent all dropped items from naturally despawning",
+	"allow all dropped items to naturally despawn"
+})
+@Since("INSERT VERSION")
+public class EffItemLifetime extends Effect {
+
+	static {
+		Skript.registerEffect(EffItemLifetime.class,
+			"prevent %itementities% from naturally despawning",
+			"prevent %itementities% from despawning naturally",
+			"allow %itementities% to naturally despawn",
+			"allow %itementities% to despawn naturally");
```

Comment:
```suggestion
			"(prevent|disallow) %itementities% from (naturally despawning|despawning naturally)",
			"allow %itementities% to (naturally despawn|despawn naturally)",
```
I think it's okay to condense these